model/manifest_test.go

```diff
@@ -31,6 +31,37 @@ func TestIsValid(t *testing.T) {
 		{"SettingSchema error", &Manifest{Id: "com.company.test", HomepageURL: "http://someurl.com", SupportURL: "http://someotherurl.com", Version: "5.10.0", MinServerVersion: "5.10.8", SettingsSchema: &PluginSettingsSchema{
 			Settings: []*PluginSetting{{Type: "Invalid"}},
 		}}, true},
+		{"Empty homepageURL, supportURL, minServerVersion, version", &Manifest{
```

Comment:
Would you please add a test with the minimum of a manifest? This manifest should only contain a valid id.